doc/tutorials/core/random_generator_and_text/random_generator_and_text.markdown

```diff
@@ -95,7 +95,7 @@ Explanation
 
        line( image, pt1, pt2, randomColor(rng), rng.uniform(1, 10), 8 );
        imshow( window_name, image );
-       if( waitKey( DELAY ) >= 0 )
+       if( (waitKey( DELAY ) & 0xFF) >= 0 )
```

Comment:
Processing looks like some animation with DELAY time between frame changes which could be immediately halted by pressing of any key.